plugins/modules/identity/keycloak/keycloak_realm_info.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_realm_info
+
+short_description: Allows obtaining Keycloak realm public information via Keycloak API
```

Comment:
```suggestion
short_description: Allows obtaining Keycloak realm public information via Keycloak API

version_added: 4.3.0
```
(Assuming this gets merged by Tuesday, otherwise more likely 4.4.0.)